src/lib/structures/Josh.ts

```diff
@@ -1678,10 +1678,7 @@ export class Josh<StoredValue = unknown> {
    * @param options The options to give all the instances.
    * @returns
    */
-  public static multi<Instances extends Record<string, Josh> = Record<string, Josh>>(
-    names: string[],
-    options: Omit<Josh.Options, 'name'> = {}
-  ): Instances {
+  public static multi<Instances extends Record<string, Josh> = Record<string, Josh>>(names: string[], options: Josh.Options = {}): Instances {
     const instances: Record<string, Josh> = {};
 
     for (const [name, instance] of names.map((name) => [name, new Josh({ ...options, name })]) as [string, Josh][]) instances[name] = instance;
```

Comment:
Can we add a provider test to make sure multiple josh instances doesn't fail

Reply:
wdym?

Reply:
So like a test in providers that just boots two with different names and makes sure the same named key with different values does return correctly